src/const/ipcGlobal.js

```diff
@@ -0,0 +1,13 @@
+export const IPC_UPDATE_AVAILABLE = 'IPC_UPDATE_AVAILABLE';
```

Comment:
the file name can be `ipcGlobalEvents.js` so while importing it becomes easier to recognise?

Reply:
These const are used both as `ipc channel names` and `ipc events` depending on usage which is why i wanted to be more lose in the naming.

For example: 
**channel name**
<img width="332" alt="image" src="https://github.com/LiskHQ/lisk-desktop/assets/8784876/20eda508-e6f5-4da8-8c29-ecbc6fddde5c">

**ipc event objcet**
<img width="295" alt="image" src="https://github.com/LiskHQ/lisk-desktop/assets/8784876/ddadd174-5700-4211-9f25-03f95562bc93">